tools/nuget/generate_nuspec_for_native_nuget.py

```diff
@@ -180,6 +182,11 @@ def generate_files(list, args):
                                        'include\\onnxruntime\\core\\providers\\cuda\\cuda_provider_factory.h') +
                           '" target="build\\native\\include" />')
 
+    files_list.append('<file src=' + '"' +
```

Comment:
openvino_provider_factory.h shouldn't be appended as a default.
note that cuda, dml etc. have conditional checks.

Reply:
i have added the condition check please review